docs/content/docs/recipes/shared-string.md

```diff
@@ -0,0 +1,454 @@
+    ```
+
+1. Add the following `import` statements. Note: `CollaborativeTextArea` and `SharedStringHelper` will be defined later.
+
+    ```js
+    import React from "react";
+    import { TinyliciousClient } from "@fluidframework/tinylicious-client";
+    import { SharedString } from "fluid-framework";
+    import { CollaborativeTextArea } from "./CollaborativeTextArea";
+    import { SharedStringHelper } from "./SharedStringHelper";
+    ```
+
+### Get Fluid Data
+
+1. The Fluid runtime will bring changes made to the text from any client to the current client. But Fluid is agnostic about the UI framework. You can use a React hook to get the Fluid data, from the SharedString object, into the view layer (the React state). Add the following code below the `import` statements. This method is called when the application loads the first time, and the value that is returned form it is assigned to a React state property.
+
+    ```js
+    const useSharedString = () => {
+
+      const [sharedString, setSharedString] = React.useState();
+      const getFluidData = async () => {
+        // TODO 1: Configure the container.
+        // TODO 2: Get the container from the Fluid service.
+        // TODO 3: Return the Fluid SharedString object.
+      }
+
+      // TODO 4: Get the Fluid Data data on app startup and store in the state
+      // TODO 5: Return the SharedString Object
+    }
+    ```
+
+1. Replace `TODO 1` with the following code.
+    ```js
+      const client = new TinyliciousClient();
+      const containerSchema = {
+        initialObjects: { sharedString: SharedString }
+      }
+    ```
+
+1. Replace `TODO 2` with the following code. Note that `containerId` is being stored on the URL hash, and if there is no `containerId` a new container is created instead.
+
+    ```js
+    let container;
+    const containerId = window.location.hash.substring(1);
+    if (!containerId) {
+      container = (await client.createContainer(containerSchema)).container;
+      const id = await container.attach();
+      window.location.hash = id;
+    } else {
+      container = (await client.getContainer(containerId, containerSchema)).container;
+    }
+    ```
+
+1. Replace `TODO 3` with the following code.
+
+    ```js
+    return container.initialObjects.sharedString;
+    ```
+
+1. Replace `TODO 4` with the following code. Note about this code:
+    - By setting an empty dependency array at the end of the `useEffect`, it is ensured that this function only gets called once.
+    - Since `setFluidSharedObjects` is a state-changing method, it will cause the React `App` component to immediately rerender.
+
+    ```js
+    React.useEffect(() => {
+      getFluidData()
+        .then(data => setSharedString(data));
+    }, []);
+    ```
+
+1. Finally, replace `TODO 5` with the following code.
+
+    ```js
+    return sharedString;
+    ```
+
+### Move the Fluid Data to the view
+
+Inside the `App()` function, add the following code. Note about this code:
```

Comment:
is it worth adding a TODO comment within the App method as a placeholder for this?